docs/chat_setup/slack_setup.md

```diff
@@ -27,9 +28,29 @@ PLUGINS_CONFIG = {
 }
 ```
 
+## Create Slack App using a Manifest
+
+1. Log in to [https://api.slack.com/apps](https://api.slack.com/apps) and select "Create New App". Select "From an app manifest."
+2. Select your preferred Slack workspace for your app.
+3. In the window titled "Enter app manifest below," copy/paste the contents of file [nautobot_slack_manifest.yml](https://github.com/nautobot/nautobot-plugin-chatops/blob/develop/setup_files/nautobot_slack_manifest.yml) in the root of this repo. Update the below settings, then click Next.
+   - On line 5, you can change the name of the Chatbot here. By default it is set to `Nautobot`
+   - On line 12, under setting `features/slash_commands/url`, update `<your-URL>` with a publicly accessible URL to your Nautobot server. Note: The trailing `/api/plugins/...` are required.
+   - Repeat this for line 31, under setting `settings/interactivity/request_url`
+4. Review the summarized settings on the next window and click Create.
+5. On the General --> Basic Information page, note the `Signing Secret` near the bottom, under App Credentials. This will be needed later for setting `SLACK_SIGNING_SECRET`.
+6. Under Distribution --> Install App, select `Install to Workspace`.
+   - If you are not a Slack admin, this step will require approval first from an Slack admin in your company.
+   - Once installed, note the `Bot User OAuth Token` here. This will be needed later for setting `SLACK_API_TOKEN`.
+7. Continue with below section "Post App-Creation Steps"
+
+> **Optional:** You can configure the App Icon on the General --> Basic Information page. Under `App Icon`, select "Choose File." You can use the supplied icon [nautobot_chatops_icon.png](https://github.com/nautobot/nautobot-plugin-chatops/blob/develop/nautobot_chatops_icon.png).
```

Comment:
The link here is giving me a 404.

Reply:
Good catch.  Updated with the correct subdirectory.

However this directory/file is part of this PR, so will give a broken link if clicked on directly until this is merged.